docs/source/changelog.rst

```diff
@@ -1,9 +1,15 @@
 =========
 Changelog
 =========
-**Future Release**
-    * Remove tensorflow and tensorhub as core requirements but instead 
-      can be installed with ``pip install nlp_primitives[complete]"``
+**v1.0.0** Aug 12, 2020
+    * Changes
+        * Remove tensorflow and tensorhub as core requirements but instead 
```

Comment:
Can this snipped mention that Universal Sentence Encoder needs these requirements?

Reply:
Updated wording.